lib/index.js

```diff
@@ -12,13 +12,15 @@ function AuthPassportLocal () {
   this.context = {};
   this.pipes = pipes;
 
-  this.init = function (config, context, isDummy) {
+  this.init = function (customConfig, context, isDummy) {
     var strategy;
+    const defaultConfig = {
+      'secret': 'changeme',
```

Comment:
I personally prefer the warning approach.

If a human need to generate a salt, he can ask to a machine to generate one, but I think that we have to let him do things